queen-of-versions/build.gradle

```diff
@@ -0,0 +1,64 @@
+apply plugin: 'com.android.library'
+apply plugin: 'com.novoda.bintray-release'
+
+ext {
+    publishedGroupId = 'co.infinum'
+    publishedArtifactName = 'queen-of-versions'
+}
+
+android {
+    compileSdkVersion buildVersions.compile
+    buildToolsVersion buildVersions.tools
+
+    defaultConfig {
+        minSdkVersion buildVersions.min
+        targetSdkVersion buildVersions.target
+        versionCode 1
+        versionName versions.prince
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation 'co.infinum:prince-of-versions:3.0.0'
```

Comment:
I believe we should use `api` instead of `implementation`.
What `implementation` does is add this dependency only to our code, `api` will do the same plus provide the dependency to whoever adds `queen`